aws/src/integration/java/org/apache/iceberg/aws/glue/TestGlueCatalogTable.java

```diff
@@ -74,8 +75,9 @@ public void testCreateTable() {
     Assert.assertTrue(response.table().parameters().containsKey(BaseMetastoreTableOperations.METADATA_LOCATION_PROP));
     Assert.assertEquals(schema.columns().size(), response.table().storageDescriptor().columns().size());
     Assert.assertEquals(partitionSpec.fields().size(), response.table().partitionKeys().size());
-    Assert.assertEquals("additionalLocations should match", tableLocationProperties.values(),
-        response.table().storageDescriptor().additionalLocations());
+    Assert.assertEquals("additionalLocations should match", tableLocationProperties.values().stream().sorted().collect(
```

Comment:
can we break newline at a more intuitive place?

```
Assert.assertEquals("additionalLocations should match", 
  tableLocationProperties.values().stream().sorted().collect(Collectors.toList()),
  response.table().storageDescriptor().additionalLocations().stream().sorted().collect(Collectors.toList()));
```

Reply:
actually, it looks like we should compare it as Set instead of List

Reply:
fixed line break. list also handles special case like duplicates.

Reply:
oh yes good point